### ISSUE TITLE
Row reorder drop indicator (DragRowArrow) should be positioned between the rows, not at the top of the row

### ISSUE THREAD

**Issue by yurigenin** [inovua/reactdatagrid#96]
- what edition are you using - enterprise
- version for `@inovua/reactdatagrid-enterprise` - latest

What you did:
When a row is dragged over other rows the drop position is indicated by the DragRowArrow component. Currently it is always rendered starting off the top of the row instead of the a more accurate position between the rows (except the last one). 

This is an example of what you see:

![image](https://user-images.githubusercontent.com/11202960/115284552-10e32a80-a10a-11eb-87ea-cb79c84fbfdd.png)

This is an example of what you want to see:

![image](https://user-images.githubusercontent.com/11202960/115284664-32441680-a10a-11eb-8daa-0766cc1e5a9e.png)

Suggested solution:

change this line:
https://github.com/inovua/reactdatagrid/blob/v4.1.7/enterprise-edition/EnterpriseColumnLayout.tsx#L625

to:

```
let boxPos;

if (index === 0) {
     boxPos = box.top;
} else if (index === ranges.length) {
     boxPos = ranges[ranges.length - 1].bottom - <drag-row-arrow-height>;
} else {
     boxPos = box.top - Math.floor(<drag-row-arrow-height> / 2);
}
```
As an additional change, it would be desirable to be able to configure the height of the insertion indicator. Currently, it is set to 3px high on this line:

https://github.com/inovua/reactdatagrid/blob/v4.1.6/enterprise-edition/plugins/row-reorder/DragRowArrow.tsx#L92

It would be much better if you allowed to make it configurable through a grid property. Then you could reference it in both places (\<drag-row-arrow-height\>).

In our application we would like to make it at least 4px high.

Thank you!

**Comment by yurigenin** [comment]
Note that I tried to use undocumented **reorderProxySize** property of the grid but it looks like it is not being used anywhere or at least not for the row reorder feature.

**Comment by inovua-admin** [comment]
FIxed in `v4.1.9`
